### PR TITLE
[WIP] Multi-Database

### PR DESCRIPTION
**Pull request by takano32**
Only use a MASTER database when the application must use it.

At other times, it reads data from multiple SLAVE databases.

### CODE REVIEW

File: airone/db_router.py [hunk 1]
```diff
@@ -15,3 +20,45 @@ def allow_relation(self, obj1, obj2, **hints):
...
+        enable = False
+        thread_id = threading.get_ident()
+        if DBSlaves.dict[thread_id] == DBType.SLAVES:
+            enable = True
+        if DBSlaves.dict[thread_id] == DBType.MASTER:
+            enable = False
+        return enable
+
+
+def airone_salve_database(func):
```
Comment (userlocalhost) [inline airone/db_router.py]: typo: salve

**Comment by takano32** [comment]
ref. https://docs.djangoproject.com/en/3.0/topics/db/multi-db/

**Comment by takano32** [comment]
To enable `DATABASE_ROUTERS`.
```
DATABASE_ROUTERS = ['airone.db_router.DBRouter']
```

**Comment by takano32** [comment]
When I asked an engineer who was familiar with Django, it seemed common to use [QuerySet.using](https://docs.djangoproject.com/en/3.0/ref/models/querysets/#using) in Django.

This PR closes because it is better to avoid unusual methods, I think.

File: airone/db_router.py (hunk 1 above)
Comment (takano32) [reply airone/db_router.py]: Thanks.

**Comment by takano32** [comment]
add some commits on `feature/multidb`

**Comment by takano32** [comment]
close.

**Pull request closed without merging**